src/main/java/propertyFiles/Bundle.properties

```diff
@@ -3062,3 +3062,31 @@ openapi.exception.invalid.format=Invalid format {0}, currently supported formats
 openapi.exception=Supported format definition not found.
 openapi.exception.unaligned=Unaligned parameters on Headers [{0}] and Request [{1}]
 
+#Users.java
+users.api.errors.bearerAuthFeatureFlagDisabled=This endpoint is only available when bearer authentication feature flag is enabled.
+users.api.errors.bearerTokenRequired=Bearer token required.
+users.api.errors.jsonParseToUserDTO=Error parsing the POSTed User json: {0}
+users.api.userRegistered=User registered.
+
+#RegisterOidcUserCommand.java
+registerOidcUserCommand.errors.userAlreadyRegisteredWithToken=User is already registered with this token.
+registerOidcUserCommand.errors.invalidFields=The provided fields are invalid for registering a new user.
+registerOidcUserCommand.errors.userShouldAcceptTerms=Terms should be accepted.
+registerOidcUserCommand.errors.provideMissingClaimsEnabled.emailAddressFieldRequired=It is required to include an emailAddress field in the request JSON for registering the user.
```

Comment:
This results in a lot of duplicate text. Can you parameterize in some way to avoid it. E.g. either have a message that 'the following fields must be provided in the (token from the identity provider, in the request JSON)' or 'the field (name) must be provided...' ?
Also - I don't think users can/should contact the identity provider - that should probably be the standard contact your Dataverse admin/support team/whatever we usually say.